backend/src/index.ts

```diff
@@ -21,5 +21,39 @@ app.delete('/api/timeslots/:id', async (req: express.Request, res: express.Respo
     })
     .catch((error) => res.status(500).json(error));
 });
+/**
+ * handle requests for flight staff
+ * handle reques for resrvation status
+*/
+app.get('/api/staff/reservation-status', async (_req: any, res: express.Response) => {
+  try {
+    const returnedTimeSlots = await Timeslot.findAll({
+      include: {
+        model: Reservation,
+        attributes: ['startTime', 'endTime', 'info'],
+      },
+    });
+    const availableSlots = returnedTimeSlots.map((obj) => (
+      {
+        ...obj,
```

Comment:
this should probably be ...obj.dataValues,
so resulting object is in correct format

Reply:
Oops!
Sorry for typo
It should be correct now